documentation/Get-Bootchart-metrics.md

```diff
@@ -0,0 +1,91 @@
+# Get Bootchart from a Clear Container #
+
+The following steps describe how to get a bootchart from a Clear Containers
+container. Currently `systemd` is the default init process in the Clear Containers mini-os
+system. The Systemd project provides a tool called `systemd-bootchart` that
+collects the CPU load, disk load, memory usage, as well as per-process information
+from the os boot process.
+
+## Configure Clear Containers to get a bootchart ##
+
+To get a bootchart from a Clear Container, follow the steps below.
+
+1. Prepare your Clear Container image
+
+The file `data/cc-bootchart.conf` can be used get a bootchart graph.
+
+The file must be installed in a Clear Container image with `systemd` and `systemd-bootchart`.
+
+The file `data/cc-bootchart` can be installed with the `configure` option
+`./autogen.sh --with-cc-image-bootchart-config-dir=DIR` where `DIR` is any of the following paths.
+
+```
+/etc/systemd/bootchart.conf
+/etc/systemd/bootchart.conf.d/*.conf
+/run/systemd/bootchart.conf.d/*.conf
+/usr/lib/systemd/bootchart.conf.d/*.conf
+```
+
+Note: All the paths above refer to paths inside the container image.
+
+2. Configure Clear Containers to boot with bootchart
+
+Make sure your kernel uses `systemd-bootchart` as the init process (PID 1).
+
+* Option 1: Using a packaged version of `cc-oci-runtime`
+
+    1. Copy the file `/usr/share/defaults/cc-oci-runtime/vm.json` to
+       `/etc/cc-oci-runtime/vm.json` (if it does not exist).
+
+    2. Replace `init=/usr/lib/systemd/systemd` to `init=PATH TO SYSTEMD-BOOTCHART BINARY`
+       (usually  `init=/usr/lib/systemd/systemd-bootchart`) in the file.
```

Comment:
Removed the only path in the case of clear containers images. the path is always init=/usr/lib/systemd/systemd-bootchart